packages/amplify-e2e-core/src/utils/add-circleci-tags.ts

```diff
@@ -53,3 +60,45 @@ export const addCircleCITags = (projectPath: string): void => {
 export function sanitizeTagValue(value: string): string {
   return value.replace(/[^ a-z0-9_.:/=+\-@]/gi, '');
 }
+
+/**
+ * Jest runs tests that start at the same time, which can lead to issues 
+ * when running amplify init on 4 tests simultaneously.
```

Comment:
This issue is caused by DynamoDb & OpenSearch simulators, which create directories that have a fixed path (not specific to each project), leading to conflicts when spinning up. These issues are exaggerated on Windows tests